tools/provision/provision.go

```diff
@@ -215,13 +215,18 @@ func Provision(conf Config) {
 	fmt.Printf("# List of channels that things can publish to\n" +
 		"# each channel is connected to each thing from things list\n")
 	for i := 0; i < conf.Num; i++ {
-		for j := 0; j < conf.Num; j++ {
+		// Creating a new routine for each connect
+		// might be heavy on the network.
```

Comment:
I would avoid fullstop on the comments

Reply:
I'm following natural language (which is, roughly speaking in this context, the definition of comments) rules. If this wasn't a full sentence, I'd leave it with no punctuation. If you insist, I'll change it, but please have in mind that this is in sync with most of the Mainflux code comments (though other comments embraced this style mostly due to GoDoc and native Go code). I prefer it this way, as well.

Reply:
OK, I leave it to you. Accepted.